wolfssl/wolfcrypt/types.h

```diff
@@ -493,6 +493,63 @@
             #endif /* _MSC_VER || __CYGWIN__ || __MINGW32__ */
         #endif /* USE_WINDOWS_API */
 
+        /* XVASPRINTF is used in ssl.c for wolfSSL_BIO_printf */
+        #ifdef __GNUC__
+            #define VASPRINTFCHECK __attribute__((format(printf, 2, 0)))
+        #else
+            #define VASPRINTFCHECK
+        #endif /* __GNUC__ */
+        #if (defined(sun) || defined(__sun) || defined(_AIX))
+            #include <stdarg.h>
+            VASPRINTFCHECK static WC_INLINE
+            int xvasprintf(char **ret, const char *format, va_list args)
+            {
+                int count;
+                char *buffer;
+                va_list copy;
+                va_copy(copy, args);
+
+                *ret = NULL;
+
+                count = vsnprintf(NULL, 0, format, args);
+                if (count >= 0)
+                {
+                    buffer = malloc(count + 1);
```

Comment:
Why normal malloc?

Reply:
XMALLOC is not fully defined here (or in `types.h`)